elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -237,6 +238,20 @@ void ElevationMapping::visibilityCleanupThread()
 void ElevationMapping::pointCloudCallback(
     const sensor_msgs::PointCloud2& rawPointCloud)
 {
+  // Check if point cloud has corresponding robot pose at the beginning
+  if(!receivedFirstMatchingPointcloudAndPose_) {
+    const double oldestPoseTime = robotPoseCache_.getOldestTime().toSec();
+    const double currentPointCloudTime = rawPointCloud.header.stamp.toSec();
+
+    if(currentPointCloudTime < oldestPoseTime) {
+      ROS_WARN_THROTTLE(5, "No corresponding point cloud and pose are found. Waiting for first match.");
+      return;
+    } else {
+      ROS_WARN("First corresponding point cloud and pose found, initialized. ");
```

Comment:
Should this be a ROS_WARN? I think warnings should only happen if something is wrong, otherwise we might end up with too many yellow warnings and the real ones would be ignored by the user.

Reply:
I agree. Thank you.